distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1988,11 +1946,47 @@ public void testCancelDownload() throws Exception {
                 "STOPPED");
     }
 
```

Comment:
Above this line there are a lot of changes that have nothing to do with your PR (except for the switch to `RestitoContent`).